examples/prometheus/README.md

```diff
@@ -116,3 +116,29 @@ Returns a running count (not a rate) of docker operations that have failed since
 > kubelet_pleg_relist_latency_microseconds
 
 Returns PLEG (pod lifecycle event generator) latency metrics.  This represents the latency experienced by calls from the kubelet to the container runtime (i.e. docker or CRI-O).  High PLEG latency is often related to disk I/O performance on the docker storage partition.
+
+### OpenShift build related queries
+
+> count(openshift_build_running_phase_start_time_seconds{} < time() - 360)
+
+Returns the number of builds that have been running for more than 10 minutes (360 seconds).
```

Comment:
60 * 10 = ? :grinning:

Reply:
LOL 

will fix, but don't want to initiate another test run given the recent announcement about the ec2 instance limit issue, so will wait to push